src/protocol/message_layer/EncryptedGroupKey.js

```diff
@@ -0,0 +1,42 @@
+import { validateIsString } from '../../utils/validations'
+
+export default class EncryptedGroupKey {
+    /**
+     * A pair (groupKeyId, encryptedGroupKey) where the encryptedGroupKey is an encrypted, hex-encoded version of the group key.
+     * @param groupKeyId
+     * @param encryptedGroupKeyHex
+     * @param serialized Optional. If given, this exact string is returned from serialize().
+     */
+    constructor(groupKeyId, encryptedGroupKeyHex, serialized = null) {
```

Comment:
Maybe these should be named parameters instead of positional for consistency with the rest of the new constructors.